src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/ConnectionTest.php

```diff
@@ -391,7 +393,7 @@ public function testGeneratedSql(AbstractPlatform $platform, string $expectedSql
     public static function providePlatformSql(): iterable
     {
         yield 'MySQL' => [
```

Comment:
DBAL 4+ separates MariaDBPlatform from MySQLPlatform. So we should add a test for it in the data provider to avoid loosing support for MariaDB.

Reply:
That test is already there, scroll down a little.